kgforge/core/forge.py

```diff
@@ -47,6 +47,7 @@
 from kgforge.core.wrappings.paths import PathsWrapper, wrap_paths
 from kgforge.specializations.mappers import DictionaryMapper
 from kgforge.specializations.mappings import DictionaryMapping
+from kgforge.specializations.databases import StoreDatabase, WebServiceDatabase
 
 
```

Comment:
you should declare _db_sources as a property, even if it's not defined on initialisation. So that it's clear all attributes the forge object may have

Reply:
Or should create_db_sources be called on forge initialisation? It seems like it isn't happening so forge doesn't have db sources